src/main/features/directions-questionnaire/forms/models/expertEvidence.ts

```diff
@@ -0,0 +1,52 @@
+import { CompletableTask } from 'models/task'
+import { ValidationErrors as GlobalValidationErrors } from 'forms/validation/validationErrors'
+import { IsDefined, IsIn, IsNotEmpty, ValidateIf } from '@hmcts/class-validator'
+import { YesNoOption } from 'models/yesNoOption'
+
+export class ValidationErrors {
+  static readonly REASON_REQUIRED: string = 'Explain what is there to examine'
```

Comment:
opinion: Would read better as `Explain what there is to examine`

Reply:
agree..updated